src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/feedback/feedback.component.ts

```diff
@@ -20,6 +22,10 @@ export class FeedbackComponent {
     description: new FormControl('', Validators.required),
   });
 
+  constructor(
+    public activeModal: NgbActiveModal
+  ) {}
+
   get f() {
```

Comment:
is this function necessary. I mean if you are checking for the required field, then you can remove these and follow what I mentioned in the above reviews.

Reply:
if you did it correctly, the output will look like this
![errorrr](https://user-images.githubusercontent.com/71764184/128175583-1248be38-994e-4583-9357-f53f0fdef565.png)